### PR TITLE
DOCSP-36869 compass auto updates

### PR DESCRIPTION
**Pull request by ltran-mdb2**
## DESCRIPTION

Updating the compass auto-update flow

## STAGING

https://preview-mongodbltranmdb2.gatsbyjs.io/compass/DOCSP-36869/upgrade/#enable-automatic-updates

## JIRA

https://jira.mongodb.org/browse/DOCSP-36869

## BUILD LOG

https://workerpool-boxgs.mongodbstitch.com/pages/job.html?collName=queue&jobId=65e7ba4311db7b0427e83fae

## Self-Review Checklist

- [x] Is this free of any warnings or errors in the RST?
- [x] Is this free of spelling errors?
- [x] Is this free of grammatical errors?
- [x] Is this free of staging / rendering issues?
- [x] Are all the links working?

## External Review Requirements

[What's expected of an external reviewer?](https://wiki.corp.mongodb.com/display/DE/Reviewing+Guidelines+for+the+MongoDB+Server+Documentation)

### CODE REVIEW

File: source/upgrade.txt [hunk 1]
```diff
@@ -47,9 +47,36 @@ dialog.
    :figwidth: 600px
    :alt: Privacy settings
 
-|compass| does not install any updates without your explicit permission.
-Instead, |compass-short| prompts you to upgrade when a new version is
-available.
+If you enable auto updates and have not downgraded |compass|,
+|compass-short| automatically updates the application four hours after
+MongoDB releases new versions. The update does not require your explicit
```
Comment (addaleax) [inline source/upgrade.txt:52]: I wouldn't mention a specific time frame here (and the 4 hours thing is not really accurate anyway)

File: source/upgrade.txt [hunk 2]
```diff
@@ -47,9 +47,36 @@ dialog.
...
+confirmation. After your machine finishes the update, |compass-short|
+shows a toast suggesting that you restart the application to apply the
```
Comment (addaleax) [inline source/upgrade.txt:54]:
```suggestion
shows a notification suggesting that you restart the application to apply the
```

I don't know if users know what a toast is, tbh.

File: source/upgrade.txt [hunk 3]
```diff
@@ -47,9 +47,36 @@ dialog.
...
+
+If you enable auto updates and have downgraded |compass| (you installed
+|compass| and then manually installed an earlier version),
+|compass-short| does not automatically update when MongoDB releases a
+new version. If a newer version is available, |compass-short| asks you
+to confirm that you want to install the update through a notification.
+You can dismiss the notification and refrain from updating. 
+
+If a new update is available and you want to update |compass-short|
+prior to the four hour delay, you can click :guilabel:`Check for
```
Comment (addaleax) [inline source/upgrade.txt:66]: Similarly here, the 4 hour things is a different timer and I just wouldn't mention it at all.

File: source/upgrade.txt (hunk 1 above)
Comment (ltran-mdb2) [reply source/upgrade.txt]: Thanks for this clarification! I removed the mentions of 4 hours

File: source/upgrade.txt (hunk 2 above)
Comment (ltran-mdb2) [reply source/upgrade.txt]: Changed to "notification"!

**Review by addaleax — APPROVED** [review]

File: source/upgrade.txt [hunk 4]
```diff
@@ -47,9 +47,36 @@ dialog.
    :figwidth: 600px
    :alt: Privacy settings
 
-|compass| does not install any updates without your explicit permission.
-Instead, |compass-short| prompts you to upgrade when a new version is
-available.
+If you enable auto updates and have not downgraded |compass|,
```
Comment (nvillahermosa-mdb) [inline source/upgrade.txt:50]:
```suggestion
If you enable auto updates and don't downgrade |compass|,
```

File: source/upgrade.txt [hunk 5]
```diff
@@ -47,9 +47,36 @@ dialog.
...
+|compass-short| automatically updates the application when MongoDB
+releases new versions. The update does not require your explicit
+confirmation. After your machine finishes the update, |compass-short|
+shows a notification suggesting that you restart the application to
+apply the update. |compass-short| does not apply the version update
+until you restart the application. 
+
+If you enable auto updates and have downgraded |compass| (you installed
+|compass| and then manually installed an earlier version),
```
Comment (nvillahermosa-mdb) [inline source/upgrade.txt:59]:
```suggestion
If you enable auto updates and subsequently downgrade |compass| by
manually installing an earlier version,
```

File: source/upgrade.txt [hunk 6]
```diff
@@ -47,9 +47,36 @@ dialog.
...
+|compass-short| does not automatically update when MongoDB releases a
+new version. If a newer version is available, |compass-short| asks you
+to confirm that you want to install the update through a notification.
+You can dismiss the notification and refrain from updating.
```
Comment (nvillahermosa-mdb) [inline source/upgrade.txt:63]:
```suggestion
You can dismiss the notification to avoid updating. 
```

File: source/upgrade.txt [hunk 7]
```diff
@@ -47,9 +47,36 @@ dialog.
...
+If a new update is available and you want to update |compass-short|
+before the automatic update, you can click :guilabel:`Check for
+updates` in the :guilabel:`MongoDB Compass` system menu. 
+
+.. figure:: /images/compass/check_for_updates.png
+  :scale: 60%
+  :alt: Check for updates location under the MongoDB Compass system menu
+
+If you click on :guilabel:`Check for updates` and an update is
+available, |compass-short| shows a notification that asks if you want to
```
Comment (nvillahermosa-mdb) [inline source/upgrade.txt:74]:
```suggestion
If an update is
available, |compass-short| shows a notification that asks if you want to
```

File: source/upgrade.txt [hunk 8]
```diff
@@ -47,9 +47,36 @@ dialog.
...
+If you enable auto updates and have downgraded |compass| (you installed
+|compass| and then manually installed an earlier version),
+|compass-short| does not automatically update when MongoDB releases a
+new version. If a newer version is available, |compass-short| asks you
+to confirm that you want to install the update through a notification.
+You can dismiss the notification and refrain from updating. 
+
+If a new update is available and you want to update |compass-short|
+before the automatic update, you can click :guilabel:`Check for
+updates` in the :guilabel:`MongoDB Compass` system menu.
```
Comment (nvillahermosa-mdb) [inline source/upgrade.txt:67]:
```suggestion
To update |compass-short| before the next automatic update, click 
:guilabel:`Check for updates` in the :guilabel:`MongoDB Compass` 
system menu:
```

File: source/upgrade.txt [hunk 9]
```diff
@@ -47,9 +47,36 @@ dialog.
...
+
+.. figure:: /images/compass/check_for_updates.png
+  :scale: 60%
+  :alt: Check for updates location under the MongoDB Compass system menu
+
+If you click on :guilabel:`Check for updates` and an update is
+available, |compass-short| shows a notification that asks if you want to
+install and restart the application immediately. 
+
+If an update is in progress, |compass-short| disables the
```
Comment (nvillahermosa-mdb) [inline source/upgrade.txt:77]:
```suggestion
While an update is in progress, |compass-short| disables the
```

**Review by nvillahermosa-mdb — APPROVED** [review]
LGTM, suggested some wording cleanup for brevity.

File: source/upgrade.txt (hunk 4 above)
Comment (ltran-mdb2) [reply source/upgrade.txt:50]: changed to "didn't downgrade" to capture past tense

**Pull request merged**